x-pack/plugins/ml/public/application/components/multi_select_picker/multi_select_picker.tsx

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiFieldSearch,
+  EuiFilterButton,
+  EuiFilterGroup,
+  EuiFilterSelectItem,
+  EuiIcon,
+  EuiPopover,
+  EuiPopoverTitle,
+  EuiSpacer,
+} from '@elastic/eui';
+import React, { FC, ReactNode, useEffect, useState } from 'react';
+
+export interface Option {
+  name?: string | ReactNode;
+  value: string;
+  checked?: 'on' | 'off';
+  disabled?: boolean;
+  type?: string;
+  onChange?: (items: any[]) => void;
+}
+
+const NoFilterItems = () => {
+  return (
+    <div className="euiFilterSelect__note">
+      <div className="euiFilterSelect__noteContent">
+        <EuiIcon type="minusInCircle" />
+        <EuiSpacer size="xs" />
+        <p>No filters found</p>
```

Comment:
This text needs internationalizing.

Reply:
Updated here https://github.com/elastic/kibana/pull/85726/commits/13fa2c8f86d766e6db81a72638ed3dd8f968cedb